Util/Legacy.php

```diff
@@ -43,7 +50,22 @@ public static function supportsFormTypeConfigureOptions()
      */
     public static function formChoicesAsValues()
```

Comment:
I'm having a hard time with this one… it's named as if it meant "we're using the new way", and used as if it meant "we're using the old way" (which would be keys as values).